conda/recipes/conda_build_config.yaml

```diff
@@ -36,7 +36,7 @@ pycifrw:
   - 4.4.1
 
 setuptools:
-  - 47.0
+  - 49.6
```

Comment:
this is inconsistent with the version in the developer environments but it's not really a problem and will be resolved when we upgrade setuptools for the python 3.10 upgrade